sklearn/metrics/_ranking.py

```diff
@@ -271,11 +271,14 @@ def _binary_uninterpolated_average_precision(
         "y_score": ["array-like"],
         "pos_label": [Real, str, "boolean", None],
         "sample_weight": ["array-like", None],
+        "drop_intermediate": ["boolean"],
     },
     prefer_skip_nested_validation=True,
 )
-def det_curve(y_true, y_score, pos_label=None, sample_weight=None):
-    """Compute error rates for different probability thresholds.
+def det_curve(
+    y_true, y_score, pos_label=None, sample_weight=None, drop_intermediate=False
+):
+    """Compute Detection Error Tradeoff (DET) for different probability thresholds.
```

Comment:
The link to the user guide below is should be to https://scikit-learn.org/stable/modules/model_evaluation.html#detection-error-tradeoff-det instead of https://scikit-learn.org/stable/visualizations.html (or to both).

Reply:
L287 below already points to https://scikit-learn.org/stable/modules/model_evaluation.html#detection-error-tradeoff-det.

Reply:
Ok then, maybe dev is already different from v1.6.